app/scripts/controllers/provisionedServices.js

```diff
@@ -0,0 +1,265 @@
+        overview.vanillaReplicaSets,
+        overview.statefulSets
+      ];
+
+      // Make sure all the binding targets have loaded first.
+      if (_.some(objectsByKind, function(collection) { return !collection; })) {
+        return;
+      }
+
+      // Build a map of pod preset selectors by binding name.
+      var podPresetSelectors = {};
+      _.each(state.bindings, function(binding) {
+        var podPresetSelector = _.get(binding, 'spec.alphaPodPresetTemplate.selector');
+        if (podPresetSelector) {
+          podPresetSelectors[binding.metadata.name] = new LabelSelector(podPresetSelector);
+        }
+      });
+
+      _.each(objectsByKind, function(collection) {
+        _.each(collection, function(apiObject) {
+          // Key by UID since name is not unique across different kinds.
+          var applicationUID = getUID(apiObject);
+
+          // Create a selector for the potential binding target to check if the
+          // pod preset covers the selector.
+          var applicationSelector = new LabelSelector(_.get(apiObject, 'spec.selector'));
+          state.bindingsByApplicationUID[applicationUID] = [];
+          state.deleteableBindingsByApplicationUID[applicationUID] = [];
+
+          // Look at each pod preset selector to see if it covers this API object selector.
+          _.each(podPresetSelectors, function(podPresetSelector, bindingName) {
+            if (podPresetSelector.covers(applicationSelector)) {
+              // Keep a map of the target UID to the binding and the binding to
+              // the target. We want to show bindings both in the "application"
+              // object rows and the service instance rows.
+              state.bindingsByApplicationUID[applicationUID].push(state.bindings[bindingName]);
+              if (!_.get(state.bindings[bindingName], 'metadata.deletionTimestamp')) {
+                state.deleteableBindingsByApplicationUID[applicationUID].push(state.bindings[bindingName]);
+              }
+              state.applicationsByBinding[bindingName] = state.applicationsByBinding[bindingName] || [];
+              state.applicationsByBinding[bindingName].push(apiObject);
+            }
+          });
+        });
+      });
+
+      $scope.bindingsByInstanceRef = _.reduce(state.bindingsByInstanceRef, function(result, bindingList, key) {
+        result[key] = _.sortBy(bindingList, function(binding) {
+          var apps =  _.get(state.applicationsByBinding, [binding.metadata.name]);
+          var firstName = _.get(_.head(apps), ['metadata', 'name']);
+          return firstName || binding.metadata.name;
+        });
+        return result;
+      }, {});*/
+    };
+
+    ProjectsService
+      .get($routeParams.project)
+      .then(_.spread(function(project, context) {
+        $scope.project = project;
+
+        var canI = $filter('canI');
+
+        if (SERVICE_CATALOG_ENABLED && canI({resource: 'instances', group: 'servicecatalog.k8s.io'}, 'watch')) {
```

Comment:
Better just to hide the nav item so you can't even get here.

I'd remove the `SERVICE_CATALOG_ENABLED` and `canI` checks, otherwise the page will be forever "Loading..." if you don't have authority. Better to see the error when the watch fails.

Same for the other `SERVICE_CATALOG_ENABLED` and `canI` checks below.